tests/utils/storageclass.go

```diff
@@ -0,0 +1,128 @@
+package utils
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	k8sv1 "k8s.io/api/storage/v1"
```

Comment:
I would call this storagev1, k8sv1 gives me impressions its the core k8s types, which it is not since corev1 are those types.

Reply:
Absolutely